docs/src/docs/asciidoc/user/107.adoc

```diff
@@ -9,25 +9,62 @@ ifdef::notBuildingForSite[]
 include::menu.adoc[]
 endif::notBuildingForSite[]
 
-== Getting started with Ehcache 3.0 & JSR-107
+== JCache overview
 
-There is nothing special for you to do to use Ehcache 3.0 as the caching provider for your application: add the
-`ehcache-3.0.0.jar` to your application's classpath (possibly removing the previous provider's jar) and you are ready to
-go:
+JCache (aka JSR-107) specification defines the standard caching API for Java.
+The specification was developed under the Java Community Process v2.9 by an expert group including members from the Ehcache developer community.
+Hence being one of the pioneers in the Java caching domain, Ehcache 3 also offers an implementation that is fully compliant with the JCache specification.
+JCache provides a very simple API set that is easy to use and vendor neutral.
+But it is not as rich and robust as the native Ehcache APIs.
```

Comment:
Intent was to promote the usage of the native API set as it offers better configurability over the JCache ones. But yeah in the context of this document which is about JSR-107 I guess that's not really needed.